diff -rux '*~' a/STM8L10x_StdPeriph_Driver/inc/stm8l10x.h b/STM8L10x_StdPeriph_Driver/inc/stm8l10x.h
--- a/STM8L10x_StdPeriph_Driver/inc/stm8l10x.h	2019-01-04 10:41:49.402506172 +0100
+++ b/STM8L10x_StdPeriph_Driver/inc/stm8l10x.h	2019-01-04 10:50:13.169914036 +0100
@@ -143,6 +143,9 @@
 #define     __O     volatile                  /*!< defines 'write only' permissions     */
 #define     __IO    volatile                  /*!< defines 'read / write' permissions   */
 
+#ifdef USE_STDINT
+#include <stdint.h>
+#else
 /*!< Signed integer types  */
 typedef   signed char     int8_t;
 typedef   signed short    int16_t;
@@ -152,6 +155,7 @@
 typedef unsigned char     uint8_t;
 typedef unsigned short    uint16_t;
 typedef unsigned long     uint32_t;
+#endif
 
 /*!< STM8L10x Standard Peripheral Library old types (maintained for legacy prupose) */
 
